devel/xxdiff/Portfile

```diff
@@ -1,62 +1,100 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
+# -*- coding: utf-8; mode: tcl; c-basic-offset: 4; indent-tabs-mode: nil; tab-width: 4; truncate-lines: t -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           qt5 1.0
 
 name                xxdiff
-version             3.2
-revision            1
+# the current xxdiff version is 5.0b1, which we approach numerically with 4.9.90
+version             4.9.90
 categories          devel
-platforms           darwin
 license             GPL-2+
-maintainers         nomaintainer
-
+platforms           darwin
+maintainers         gmail.com:rjvbertin openmaintainer
 description         xxdiff is a graphical merging tool
 long_description    xxdiff is a graphical browser for viewing the \
-                    differences between two or three files, or between \
-                    two directories, and can be used to produce a \
-                    merged version.
+                    differences between two or three files, or between two \
+                    directories, and can be used to produce a merged version. \
+                    It is inspired by the venerable SGI xdiff utility
+
 homepage            http://furius.ca/xxdiff/
 
-depends_lib         port:bison \
-                    port:flex \
-                    port:qt3 \
-                    port:libGLU
 
-# xxdiff links with libGLU if it is installed; so make sure it is
-# installed already.  See also: https://trac.macports.org/ticket/47693
+depends_lib         port:bison \
+                    port:flex
 
-master_sites        sourceforge:project/xxdiff/xxdiff/${version}/
-use_bzip2           yes
-checksums           md5     301ff399a1e4be8b68c38c3e27c0b1ad \
-                    sha1    ab2848fd9279f55b5db270f319ad4791154ecf82 \
-                    rmd160  12e7527f39713b925f05b687c2ad8b60459bb670
+fetch.type          hg
+hg.url              https://bitbucket.org/blais/xxdiff
+hg.tag              815d97a7a0a1
+distname            ${name}-5.0
+worksrcdir          ${distname}
 
-# https://trac.macports.org/ticket/41600#comment:47
-patchfiles          bison3.patch
+patchfiles-append   patch-xxdiffpro.diff
 
-# "Configure" and build adapted from README.build.
+set worksrcpath     ${workpath}/${worksrcdir}
+configure.pre_args  -f ${worksrcpath}/src/Makefile.bootstrap \
+                     MAKEDIR=${worksrcpath}/src
+configure.dir       ${workpath}/build
+configure.cmd       make
+configure.pre_args-append \
+                    QMAKE=${qt_qmake_cmd} QMAKEOPTS="-spec ${qt_qmake_spec}"
+build.dir           ${configure.dir}
 
-build.type          gnu
+# MacPorts base can add -stdlib=libc++ to configure.cxxflags, in which case it also has
+# to be added to configure.ldflags
+if {![string match "*stdc=libc++*" ${configure.cxxflags}]} {
```

Comment:
Do you mean `-stdlib`? Why not simply:

    configure.ldflags-append -stdlib=${configure.cxx_stdlib}